homecode-pipeline/homecode-pipeline.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+import kfp
+
+search_path = os.path.dirname(os.path.abspath(__file__)) + "/../components"
+component_store = kfp.components.ComponentStore(local_search_paths=[search_path])
+
+dkube_training_op = component_store.load_component("training")
+
+username = (
+    os.getenv("username") if os.getenv("username") != None else os.getenv("USERNAME")
+)
+
```

Comment:
any issue in passing username as argument in the pipeline ?

Reply:
i have added user as argument.